packages/angular/build/tools/baseline_browserslist/generate_browserslist.mts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import { getCompatibleVersions } from 'baseline-browser-mapping';
+
+// Map `baseline-browser-mapping` browsers to `browserslist` browsers.
+const browsers = new Map(
```

Comment:
OOC: why not use a Record? seems like there is no real reason why we need a Map here.